pulsar-transformations/pom.xml

```diff
@@ -30,6 +30,38 @@
     <build.dir>${project.build.directory}</build.dir>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>ai.djl</groupId>
+      <artifactId>api</artifactId>
+      <version>${djl.version}</version>
```

Comment:
please set this as "provided" and add the dependency in the new pulsar-ai-tools module.
Basically now we don't want the new third party dependencies to be bundled into the "pulsar-transformations" nar file but only in the ai-tools nar.